swan-lake/learn/using-the-cli-tools.md

```diff
@@ -155,41 +155,40 @@ These everyday commands are your best friends! They address the very basics of p
 </tr>
 </table>
 
-## Module Commands
+## Package Commands
 
-These commands allow you to work with the Ballerina Central to share Ballerina modules with others in a safe, secure, and dependable way.
+Ballerina packages are the way to organize real-world Ballerina development tasks. Some of them also allow you to work with the Ballerina Central to share Ballerina modules with others in a safe, secure, and dependable way.
 
 <table class="cComandTable">
 <tr>
-<td class="cCommand">pull</td>
-<td class="cDescription">Pull a module from Ballerina Central.
+<td class="cCommand">new</td>
+<td class="cDescription">Create a Ballerina package. For more information, see <a href="/swan-lake/learn/structuring
+-ballerina-code">Structuring Ballerina Code</a>.
 </td>
 </tr>
 <tr>
-<td class="cCommand">push</td>
-<td class="cDescription">Upload a module to Ballerina Central. For more information, see <a href="/swan-lake/learn/publishing-modules-to-ballerina-central">Publishing Modules to Ballerina Central</a>.
+<td class="cCommand">init</td>
+<td class="cDescription">Create a new Ballerina package in the current directory.
 </td>
 </tr>
 <tr>
-<td class="cCommand">search</td>
-<td class="cDescription">Search Ballerina Central for modules.
+<td class="cCommand">add</td>
+<td class="cDescription">Create a new Ballerina module in a package.
 </td>
 </tr>
-</table>
-
-## Project Commands
-
-Ballerina projects are the way to organize real-world Ballerina development tasks. 
-
-<table class="cComandTable">
 <tr>
-<td class="cCommand">new</td>
-<td class="cDescription">Create a Ballerina project. For more information, see <a href="/swan-lake/learn/structuring-ballerina-code">Structuring Ballerina Code</a>.
+<td class="cCommand">pull</td>
+<td class="cDescription">Pull a module from Ballerina Central.
 </td>
 </tr>
 <tr>
-<td class="cCommand">add</td>
-<td class="cDescription">Create a new Ballerina module in a project. For more information, see <a href="/swan-lake/learn/structuring-ballerina-code">Structuring Ballerina Code</a>.
+<td class="cCommand">push</td>
+<td class="cDescription">Upload a module to Ballerina Central. For more information, see <a href="/swan-lake/learn/publishing-modules-to-ballerina-central">Publishing Modules to Ballerina Central</a>.
```

Comment:
Ballerina Central works with packages, not modules. Please change the other places accordingly.

Reply:
No need to change the url it will be changed in [this PR](https://github.com/ballerina-platform/ballerina-dev-website/pull/1714).

Reply:
Fixed.